javascript-bank/bank.js

```diff
@@ -0,0 +1,55 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+  this.bank = { nextAccountNumber: this.nextAccountNumber, accounts: this.accounts };
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  this.balance = balance;
+  this.holder = holder;
+  this.newAccount = [];
```

Comment:
Please do not add additional properties not requested in the instructions. You can make local variables to the function to get the same effect without polluting the object the data is being stored in.